### PR TITLE
Update changelog with sprint updates

### PR DESCRIPTION
**Pull request by JPrause**
Updates are for 61 through 66

### CODE REVIEW

**Comment by JPrause** [comment]
@chriskacerguis I needed to practice creating changelogs. How does this look? If okay, I'll remove the wip.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
+### Changes for Sprint 66
```
Comment (AllenBW) [inline CHANGELOG.md]: @JPrause we jump from three pounds to two pounds to three pounds, on purpose? justtttt wonderrrringggg 🤔

Comment (JPrause) [inline CHANGELOG.md]: Seems my fingers got a little happy. Let me update to be consistent.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+- Updated code owners for project [(#866)](https://github.com/ManageIQ/manageiq-ui-service/pull/866)
+- Removed ace component from SUI [(#863)](https://github.com/ManageIQ/manageiq-ui-service/pull/863)
+- Update code owners to be formatted correctly [(#859)](https://github.com/ManageIQ/manageiq-ui-service/pull/859)
+- Update code owners [(#857)](https://github.com/ManageIQ/manageiq-ui-service/pull/857)
+- Fixed: the VM status shows "retired" for all VM's
```
Comment (AllenBW) [inline CHANGELOG.md]: https://github.com/ManageIQ/manageiq-ui-service/pull/856

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+- Adds font fabulous dep [(#854)](https://github.com/ManageIQ/manageiq-ui-service/pull/854)
+
+### Changes for Sprint 65
+- Display colored fonticons before custom buttons [(#851)](https://github.com/ManageIQ/manageiq-ui-service/pull/851)
+- BZ#142772825 - snapshottimeline [(#848)](https://github.com/ManageIQ/manageiq-ui-service/pull/848)
+- BZ#1469802 - Buttons are updated on power operations
```
Comment (AllenBW) [inline CHANGELOG.md]: https://github.com/ManageIQ/manageiq-ui-service/pull/844

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+- Remove Saucelabs from Travis CI [(#843)](https://github.com/ManageIQ/manageiq-ui-service/pull/843)
+- BZ 1467569- in the self service portal after a little time displaying a vm
```
Comment (AllenBW) [inline CHANGELOG.md]: https://github.com/ManageIQ/manageiq-ui-service/pull/842

File: CHANGELOG.md (hunk 2 above)
Comment (JPrause) [inline CHANGELOG.md]: Thanks for reviewing @AllenBW

**Pull request merged**